scripts/sct_warp_template.py

```diff
@@ -248,5 +252,4 @@ def main(args=None):
 
 if __name__ == "__main__":
     sct.init_sct()
-    param = Param()
```

Comment:
You can keep the param_default in the module scope and avoid passing a useless/cryptic argument.

Reply:
I accepted your suggestion.
My only concern is that when instantiating the class `WarpTemplate`, the parameters below will not be accessible through the constructor and therefore can't be changed from what's in the `Param` class definition. 
```
        self.folder_template = param.folder_template
        self.folder_atlas = param.folder_atlas
        self.folder_spinal_levels = param.folder_spinal_levels
```

But that seems to be the desired behavior anyway.